core/domain/stats_services.py

```diff
@@ -122,6 +169,14 @@ def record_answers(
         submitted_answer_list):
     """Optimally record a group of answers using an already loaded exploration..
     The submitted_answer_list is a list of SubmittedAnswer domain objects.
+
+    Args:
+        exploration_id: str. The exploration ID.
+        exploration_version: int. The version of the exploration.
```

Comment:
After you said I went inside the file core/controllers/admin.py and searched for "get_state_answers" I couldn't find it. And I was not aware that I can use test files for this purpose.